File: selectSort/select_sort.go
package main

import "fmt"

func main(){
	data := []int{2, 1, 4, 3}
	len_data := len(data)
    for i := 0; i < len_data; i++{
		tmp := i
		for j := 0; j < len_data-1 ; j++{
			if data[j] > data[tmp]{
				tmp = j
			}
		data[i], data[tmp] = data[tmp], data[i]
		}
	} 
	fmt.Println(data)
}

File: Algorithm/sort/selectSort/select_sort.py
data=[3,1,4,2]

for i in range(len(data)):
  tmp = i
  for j in range(i+1,len(data)):
    if data[j] < data[tmp]:
      tmp=j
  data[i],data[tmp] = data[tmp],data[i]

print(data)
